### PR TITLE
docs: point to "firebase.google" domain for hyperlinks in the usage section of `README.md` files (for the missing packages)

### PR DESCRIPTION
**Pull request by nilsreichardt**
## Description

https://github.com/firebase/flutterfire/pull/8814 was a draft pull request because it was just a proposal for a few packages 🙈

> This PR is a proposal and should show a proof of concept for the new format. If this is a good solution, I will continue adding the links for the other packages.

This PR adds the links for the other packages.

cc: @russellwheatley 

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/firebase_analytics/firebase_analytics/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ To get started with Firebase Analytics for Flutter, please [see the documentatio
 
 ## Usage
 
-To use this plugin, please visit the [Analytics Usage documentation](https://firebase.flutter.dev/docs/analytics/usage)
+To use this plugin, please visit the [Analytics Usage documentation](https://firebase.google.com/docs/firestore/quickstart#add_data)
```
Comment (russellwheatley) [inline packages/firebase_analytics/firebase_analytics/README.md:15]:
```suggestion
To use this plugin, please visit the [Analytics Usage documentation](https://firebase.google.com/docs/analytics/events?platform=flutter)
```

File: packages/cloud_functions/cloud_functions/README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ To get started with Cloud Functions for Firebase, please [see the documentation]
 
 ## Usage
 
-To use this plugin, please visit the [Functions Usage documentation](https://firebase.flutter.dev/docs/functions/usage)
+To use this plugin, please visit the [Functions Usage documentation](https://firebase.google.com/docs/functions/callable#call_the_function)
```
Comment (russellwheatley) [inline packages/cloud_functions/cloud_functions/README.md:16]:
```suggestion
To use this plugin, please visit the [Functions Usage documentation](https://firebase.google.com/docs/functions/callable#dart)
```

**Review by russellwheatley — CHANGES_REQUESTED** [review]
Thanks, @nilsreichardt. A few updates and it LGTM.

File: packages/firebase_analytics/firebase_analytics/README.md (hunk 1 above)
Comment (nilsreichardt) [reply packages/firebase_analytics/firebase_analytics/README.md:15]: omg, thanks

**Review by russellwheatley — APPROVED** [review]

**Review by Lyokone — APPROVED** [review]

**Pull request merged**